### DISCUSSION TITLE
Where can I download the app?

### DISCUSSION THREAD

**Discussion by senolfndk** [Arlodotexe/strix-music#136] [Q&A] [answered]
where can i download the app

**Comment by Arlodotexe** [comment, marked as answer]
As pointed out in the readme, we're still in the alpha stage and the app isn't ready for everyone to use.
Patreons and code contributors will continue to get access to the alpha builds in the Microsoft Store.

**Reply by senolfndk** [reply]
ok

**Reply by sojab0on** [reply]
i am a patreon subscriber and pay monthly but i cant get to the alpha builds still
